src/test-utils/luma.gl/headless.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable quotes, no-console */
+/* global console */
+import 'luma.gl';
+import {global, isBrowser} from './utils/globals';
```

Comment:
what's the purpose of luma.gl folder under the test-utils folder?

Reply:
> what's the purpose of luma.gl folder under the test-utils folder?

I am going to make a lot of changes to some parts of luma.gl: 
* IO (support for node+browser load/save)
* headless-gl initialization. 

I am pretty sure there will be a number of iterations until I get it right, so for efficiency, I will temporarily copy files here, and then move them back to luma.gl in a big PR, rather than publish alpha releases for luma.gl etc.